time_tracker_api/time_entries/time_entries_model.py

```diff
@@ -74,6 +75,14 @@ def __init__(self, data):  # pragma: no cover
     def running(self):
         return self.end_date is None
 
+    def __add__(self, other):
```

Comment:
I understand this method can be used to replicate data of projects into a time-entry payload. But I don't see that is being used in the changes of the PR. If you expect to use in a future PR, a #TODO  comment would be very nice to have.